WalletWasabi.Gui/Controls/WalletExplorer/CoinViewModel.cs

```diff
@@ -68,7 +68,11 @@ public void SubscribeEvents()
 
 				this.WhenAnyValue(x => x.Confirmed, x => x.CoinJoinInProgress, x => x.Confirmations)
 					.ObserveOn(RxApp.MainThreadScheduler)
-					.Subscribe(_ => RefreshSmartCoinStatus());
+					.Subscribe(_ => 
+					{
+						this.RaisePropertyChanged(nameof(Confirmations));
```

Comment:
I wonder if calling PropChanged here is okay? it may cause a stackoverflow by looping if im not mistaken?

Reply:
Yes this is not OK. It will cause a deadlock on osx.

Reply:
This.